src/dart/ast.rs

```diff
@@ -512,16 +754,23 @@ impl Default for FnSig {
     }
 }
 
+/// The body of a function.
 #[derive(Debug)]
 pub enum FnBody {
+    /// The body is an expression preceded by a fat arrow (`=> expr`).
```

Comment:
I say "fat" when talking to disambiguate but here "arrow" is enough. Should maybe mention that the expression's value is returned.

Reply:
They use it in the Dart documentation as well, fwiw. ^_^

Reply:
"Maybe" is a bit lax.

Reply:
Oh then "fat arrow" is fine.